agenda-webapps/src/main/webapp/vue-app/exchange-connector/agendaExchangeConnector.js

```diff
@@ -21,7 +21,7 @@ export default {
   isOauth: false,
   canConnect: true,
   canPush: false,
-  initialized: false,
+  initialized: true,
```

Comment:
this modif should be done after exchange connector moving to agenda-connectors codebase

Reply:
PR to be closed IMHO and same modif to be done in agenda-connectors codebase instead